### PR TITLE
Fixes: set_contracts_state

### PR DESCRIPTION
**Pull request by Buckram123**
Small fix to ensure state is patched(also refactored confusing nesting) and that we don't try to use chain_name in path

### CODE REVIEW

File: cw-orch-daemon/src/state.rs [hunk 1]
```diff
@@ -357,20 +357,13 @@ pub trait DeployedChains<Chain: CwEnv>: cw_orch_core::contract::Deploy<Chain> {
     /// Set the default contract state for a contract, so that users can retrieve it in their application when importing the library
     /// If a state is provided, it is used for all contracts, otherwise, the state is loaded from the crate's state file.
     fn set_contracts_state(&mut self, custom_state: Option<Value>) {
-        let state;
-
-        let state_file = Self::deployed_state_file_path();
-        if let Some(custom_state) = custom_state {
-            state = custom_state;
```
Comment (Buckram123) [inline cw-orch-daemon/src/state.rs:364]: It was not patched for custom

File: cw-orch-daemon/src/state.rs [hunk 2]
```diff
@@ -381,8 +374,6 @@ pub trait DeployedChains<Chain: CwEnv>: cw_orch_core::contract::Deploy<Chain> {
             // We try to get the code_id for the contract
             if contract.code_id().is_err() {
                 let code_id = state
-                    .get(env_info.chain_name.clone())
-                    .unwrap_or(&Value::Null)
```
Comment (Buckram123) [inline cw-orch-daemon/src/state.rs:385]: Those won't be here after patch

**Comment by Kayanski** [comment]
Nice catch, do you need a patch to the released version ?

**Comment by Buckram123** [comment]
> Nice catch, do you need a patch to the released version ?

No, turned out I was debugging wrong thing, haha. Thank you!

**Pull request merged**